release/RELEASE.md

```diff
@@ -1,14 +1,17 @@
 # Making a Release
 
-1. Make a PR that only bumps the version number in the VERSION
+1. Update https://github.com/digital-asset/daml/docs/support/release-notes.rst
+   by adding a new header for the new version above any changes since the last
+   version.
+2. Make a PR that only bumps the version number in the VERSION
    file. It is important that the PR only changes the VERSION file.
-2. Merge the PR using a rebase merge.
```

Comment:
You can write the numeration with all `1`s, so the renderer does the numeration automatically:
```markdown
1. foo
1. bar
1. baz
```

Reply:
I thought about it, but then it looks bad in raw. Would you strongly prefer all 1.s?

Reply:
Ah no, but given that github renders the markdown file whenever possible, I don't think it's too bad. It's just easier for future "refactoring" of the list :)